src/main/java/ua/rent/masters/easystay/controller/BookingController.java

```diff
@@ -31,26 +34,32 @@ public class BookingController {
 
     @PostMapping
     @ResponseStatus(HttpStatus.CREATED)
+    @PreAuthorize("hasAnyRole('ROLE_ADMIN','ROLE_MANAGER','ROLE_USER')")
     public BookingResponseDto createNewBooking(
             @RequestBody @Valid BookingRequestDto requestDto) {
         return bookingService.create(requestDto);
     }
 
     @GetMapping("/my")
     @ResponseStatus(HttpStatus.OK)
-    public List<BookingResponseDto> findAllUserBooking(Pageable pageable) {
-        return bookingService.getAll(pageable);
+    @PreAuthorize("hasAnyRole('ROLE_USER')")
+    public List<BookingResponseDto> findAllUserBooking(
+            @AuthenticationPrincipal User user,
+            Pageable pageable) {
+        return bookingService.getAll(user.getId(),pageable);
```

Comment:
```suggestion
        return bookingService.getAll(user.getId(), pageable);
```
never knew it can be without space (checkstyle)